index/scorch/snapshot_index_vr.go

```diff
@@ -28,6 +28,8 @@ import (
 	segment_api "github.com/blevesearch/scorch_segment_api/v2"
 )
 
+const VectorSeachSupportedSegmentVersion = 16
```

Comment:
Do you think this should be exported from zapx instead?
Something like this - https://github.com/blevesearch/zapx/blob/86b116167ea9755282d9a830f5d159cd0b3b857a/build.go#L27 .

Maybe it could be titled MinVectorSearchSupportedVersion since we may release newer zapx versions, all of which will support vector search and this will just be the minimum.

Just my $0.02. What do you think?

Reply:
Actually I think scorch_segment_api is the best place for it

Reply:
Neither zapx nor scorch_segment_api feel like the right place for this.
Let's leave this here and get the check in place for now. 

I think this naming and the check should be ok when we introduce later zapx versions (whenever that is) also.